src/protocol.cpp

```diff
@@ -86,11 +80,6 @@ static const char* ppszTypeName[] =
     NetMsgType::SPORK,
     NetMsgType::MASTERNODEPAYMENTVOTE,
     NetMsgType::MASTERNODEPAYMENTBLOCK, // reusing, was MNSCANERROR previousely, was NOT used in 12.0, we need this for inv
-    NetMsgType::MNBUDGETVOTE, // deprecated since 12.1
-    NetMsgType::MNBUDGETPROPOSAL, // deprecated since 12.1
-    NetMsgType::MNBUDGETFINAL, // deprecated since 12.1
-    NetMsgType::MNBUDGETFINALVOTE, // deprecated since 12.1
-    NetMsgType::MNQUORUM, // not implemented
```

Comment:
Can't do this, read the `NOTE` (line 83) ;)

>NOTE: include non-implmented here, we must keep this list in sync with enum in protocol.h

Reply:
Yep, I see your point (and commented more below), but the fact that `we must keep this list in sync with enum in protocol.h` seems like a code smell to me. I'm also not convinced that an `enum` is actually the right data type, since we're just directly assigning values _and_ they're part of the protocol. Really I think that's a poor case for an enum, since the values have to be maintained.